Roles/Player/README.md

```diff
@@ -23,3 +23,4 @@
 * [@flaxscrip](https://flaxscrip.github.io/flaxscrip-space/)
 * [@dawnTestCode](https://dawntestcode.github.io/dawnTestCode-space//)
 * [@genecyber](https://genecyber.github.io/ShannonCodePlayer-space/)
+* [@angelabacca](https://github.com/angelabacca/angela-player-space/)
```

Comment:
This should be a github.io link

Reply:
I'll reach out to Angela on Slack. She also needs to modify the player-space content with her personal information.

Reply:
@angelabacca published her player-space using Gitpages. I will update the README.md. https://angelabacca.github.io/angela-player-space/